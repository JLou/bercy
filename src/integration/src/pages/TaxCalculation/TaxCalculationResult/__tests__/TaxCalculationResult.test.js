import React from 'react';
import { render } from '@testing-library/react';
import { TaxCalculationResult } from '../TaxCalculationResult';

describe('TaxCalculationResultResult', () => {
  it('Renders TaxCalculationResult component without crashing', () => {
    const { asFragment } = render(<TaxCalculationResult />);
    expect(asFragment()).toMatchSnapshot();
  });
});
