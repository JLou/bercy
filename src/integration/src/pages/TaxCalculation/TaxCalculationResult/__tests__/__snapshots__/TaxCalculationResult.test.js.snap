// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaxCalculationResultResult Renders TaxCalculationResult component without crashing 1`] = `
<DocumentFragment>
  <h1
    class="af-title--content"
  >
    Résultat
  </h1>
  <div
    class="tax-result"
  >
    <dl
      class="af-restitution__listdef"
    >
      <dt
        class="af-restitution__listdef-item"
      >
        <span
          class="af-restitution__text"
        >
          Taux d'imposition
        </span>
      </dt>
      <dd
        class="af-restitution__listdef-value"
      >
        ***
      </dd>
    </dl>
    <dl
      class="af-restitution__listdef"
    >
      <dt
        class="af-restitution__listdef-item"
      >
        <span
          class="af-restitution__text"
        >
          Montant impôt
        </span>
      </dt>
      <dd
        class="af-restitution__listdef-value"
      >
        ****
      </dd>
    </dl>
    <dl
      class="af-restitution__listdef"
    >
      <dt
        class="af-restitution__listdef-item"
      >
        <span
          class="af-restitution__text"
        >
          Nombre de part
        </span>
      </dt>
      <dd
        class="af-restitution__listdef-value"
      >
        ****
      </dd>
    </dl>
  </div>
</DocumentFragment>
`;
